### PR TITLE
Scalpels now cut like knives

### PR DESCRIPTION
**Pull request by beck-thompson**
## About the PR

Title!

## Why / Balance

Was brought up in relation to plushies but it makes sense they have the ability in general!

## Media

image

I spent so long trying to record but OBS has decided not to work.

**Changelog**

:cl: Beck Thompson
- add: Scalpels and shivs now work as knives!

### CODE REVIEW

**Comment by thebadman4662** [comment]
Does it fix issue where scalpels can't make baseball bat(but glass shard can?) 😆

**Comment by lzk228** [comment]
i would remove it from glass shard, so you have to put some effort in it and make a shiv from it, which you can use as a knife

**Comment by beck-thompson** [comment]
Makes sense I agree. Forgot about the shiv!

File: Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml [hunk 1]
```diff
@@ -68,6 +68,10 @@
   components:
   - type: Sharp
     butcherDelayModifier: 1.5 # Butchering with a scalpel, regardless of the type, will take 50% longer
+  - type: Tool
+    qualities:
+      - Slicing
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml:73]:
```suggestion
    qualities:
    - Slicing
```
indentation

**Review by slarticodefast — APPROVED** [review]

**Comment by slarticodefast** [comment]
This will also extend to shivs because they inherit form scalpels, but that is good in my opinion.
You might want to mention that in the changelog.

**Comment by chromiumboy** [comment]
Looks good to me, tested in-game fine

Updated your comment to include shivs

**Comment by beck-thompson** [comment]
Thank you I forgot to respond and fix it (Midterms 😭) ❤️

**Pull request merged**